changelogs/changelog.yml

```diff
@@ -1,5 +1,62 @@
 ancestor: null
 releases:
+  2.3:
+    changes:
+      major_changes:
+      - Add support Pardus 21.x autoinstall
+      - Add support FreeBSD 13 or later autoinstall
+      - Add support Ubuntu 23.04 desktop autoinstall and test
+      - Update log plugin to support more than one same test case runs
+      minor_changes:
+      - Enable Debian online repo for installing open-vm-tools dependencies
+      - Fix Photon OS 5.0 testing issues
+      - Update known issue for Windows NVMe hot extend which has been fixed in ESXi 8.0 U1
+      - Add hardwarwe version check for nvme 1.3
+      - Set VM primary NIC MAC address at "env_setup" or "deploy_vm"
+      - Upgrade cloud-init in Oracle Linux 8.7 and 9.1 for cloud-init GOSC
+      - Replace "vars_files" with "include_vars" task in playbooks
+      - Change the VMware Tools version for known issue in Windows OS in-place upgrade
+      - Update cloud-init GOSC complete message when cloud-init version >= 23.1
+      - Update "vmtools_url_path" and "vmtools_iso_path" with VMware Tools 12.2.0 path
+      - Remove messages about redirecting modules and deprecation warnings
+      - Add handling VBS known issue on ESXi 7.0.3
+      - Collect Linux OVA configs at OVA deployment
+      - Add check driver version in Windows test case "check_inbox_driver"
+      - Improve "check_os_fullname" test case to check guest id and guest family
+      - Skip test case "secureboot_enable_disable" when VBS is enabled
+      - Enable WSL testing for Windows Server
+      bugfixes:
+      - Move "check_vm_settings.yml" after reverting base snapshot
+      - Fix variables undefined error during deploying VM
+      - Move adding KMS server task at the beginning of deploy_vm.yml
+      - Fix getting files under recursively directory on Windows
```

Comment:
"recursively"-->"recursive"

Reply:
Move recursively after directory, which is more sensible.